src/docs/api/authentication/guide_authenticate-to-the-laserfiche-api/index.md

```diff
@@ -16,6 +16,9 @@ See LICENSE-DOCUMENTATION and LICENSE-CODE in the project root for license infor
 
 Create a connection to Laserfiche services to begin using the Laserfiche API. Subsequent requests can use the access token received in the response from the initial connection creation request.
 
+{: .note }
+Note: To prevent CSRF attacks, the resulting Access Token should NOT be stored as a cookie. The access token should be stored securely and only accessible by the applicable user.

```

Comment:
Access Tokens used to access APIs are secrets and should be stored securely. To prevent CSRF attacks, it's NOT recommended to store secrets or sensitive information in cookies.